build.sbt

```diff
@@ -20,19 +20,22 @@ libraryDependencies ++= Seq(
   cache,
   javaWs,
   javaJpa,
-  "org.eclipse.persistence" % "eclipselink" % "2.6.2",
-   "com.github.ONSdigital" % "dp-dd-backend-model" % "1.0.29",
-//  "uk.co.onsdigital.discovery" % "dd-model" % "1.0.29",
+//   "com.github.ONSdigital" % "dp-dd-backend-model" % "1.0.29",
+  "uk.co.onsdigital.discovery" % "dd-model" % "1.0.34",
```

Comment:
Revert back to using published version when merged. Also, I believe it would be `1.0.35` once you merge your PR for the model.